### PR TITLE
update install instructions for people who do not have nvm or npm installed

### PR DESCRIPTION
**Pull request by DaniBodor**
These are the instructions I just followed. They were found [here](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#using-a-node-version-manager-to-install-nodejs-and-npm) and [here](https://github.com/nvm-sh/nvm/blob/master/README.md), but it was a bit of a hassle to delve through the links to find the specific instructions required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,16 +37,39 @@ git clone git@github.com:esciencecenter-digital-skills/NEBULA.git
...
+
+Now, install node:
+```bash
+nvm install node
+```
+
+Finally, you can install the node package manager itself. Make sure you install it in the content repository directory
+```bash
+cd /path/to/your/content/repository
+npm install -g npm
```
Comment (DaniBodor) [inline README.md:72]: remind me whether this needed to be in the content directory or in the nebula directory...

Comment (JaroCamphuijsen) [reply README.md:72]: npm install needs to be run in the NEBULA directory

Comment (JaroCamphuijsen) [inline README.md:72]:
```suggestion
npm install
```

**Review by JaroCamphuijsen — CHANGES_REQUESTED** [review]
I'm thinking we might want to move the instructions for installing nvm, node and npm to the NEBULA documentation instead of this README. But if you apply these changes, I'm happy to merge it here and then later move it to NEBULA if we think that is a better place.

**Comment by DaniBodor** [comment]
> I'm thinking we might want to move the instructions for installing nvm, node and npm to the NEBULA documentation instead of this README. But if you apply these changes, I'm happy to merge it here and then later move it to NEBULA if we think that is a better place.

I agree it would be best to have the information NEBULA repo. Feel free to go ahead as you suggested to merge here (I accepted all your suggestions) and then transfer it or immediately move the instructions there, whichever you find more convenient.

**Review by JaroCamphuijsen — APPROVED** [review]

**Pull request merged**